lib/trento/application/usecases/clusters/clusters.ex

```diff
@@ -129,7 +129,7 @@ defmodule Trento.Clusters do
     |> select_merge([c, e], %{cib_last_written: e.cib_last_written})
   end
 
-  defp maybe_request_checks_execution(%{selected_checks: []}), do: :ok
+  defp maybe_request_checks_execution(%{selected_checks: []}), do: {:error, :no_checks_selected}
```

Comment:
Now, on an empty check selection the `ClusterController.request_checks_execution` operation returns 422 instead of 202, which is correct, desirable and still covered by the endpoint's openapi spec (falling in the 422 case)

I cannot but think, though, that the very same endpoint might start suddenly returning 422s where it before returned 202s and as a client being integrated it might be annoying.

What about tracking and deferring this? 
The `ClusterController.request_checks_execution` operation needs anyway an adjustment (for instance, it would never return a bad_request as defined in the spec)

We could introduce this change and bump the version of the endpoint, if needed.

wdyt?

Reply:
Sure let's tackle it at another time, we may refactor it. I was just thinking about keeping the host and cluster checks execution similar.
I will create a new ticket for this!